secrets/secrets.yml

```diff
@@ -0,0 +1,221 @@
+        default: Staging S3 object key
+      InfrastructureConfigProductionKey:
+        default: Production S3 object key
+Parameters:
+  InfrastructureStorageStackName:
+    Default: infrastructure-storage
+    Description: The name of a previously launched infrastructure storage stack
+    Type: String
+  NotificationsStackName:
+    Default: notifications
+    Description: The name of a previously launched notifications stack
+    Type: String
+  InfrastructureConfigStagingKey:
+    Default: template-config-staging.zip
+    Description: >
+      The S3 object key for the staging template configuration file archive
+      (.zip), inside the InfrastructureConfigBucket
+    Type: String
+  InfrastructureConfigProductionKey:
+    Default: template-config-production.zip
+    Description: >
+      The S3 object key for the production template configuration file archive
+      (.zip), inside the InfrastructureConfigBucket
+    Type: String
+Resources:
+  SecretsIAMGroup:
+    Type: "AWS::IAM::Group"
+    Properties:
+      GroupName: !Sub ${InfrastructureStorageStackName}-manage-secrets
+      Path: "/"
+      Policies:
+        - PolicyName: !Sub ${InfrastructureStorageStackName}-encrypt-secrets
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Id: !Sub key-${InfrastructureStorageStackName}-secrets-encrypt-policy
+                Effect: Allow
+                Action:
+                  - kms:Encrypt
+                  - kms:Decrypt
+                Resource:
+                  - !GetAtt SecretsKey.Arn
+        - PolicyName: !Sub ${InfrastructureStorageStackName}-secrets-s3-read-write
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Id: !Sub key-${InfrastructureStorageStackName}-secrets-group-access-policy
+                Effect: Allow
+                Action:
+                  - s3:GetObject
+                  - s3:GetObjectVersion
+                  - s3:PutObject
+                Resource:
+                  - !Sub arn:aws:s3:::${InfrastructureStorageStackName}-secrets/*
+  SecretsInstanceDecryptPolicy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyName: !Sub key-${InfrastructureStorageStackName}-secrets-instance-decrypt-policy
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - kms:Decrypt
+            Resource:
+              - !GetAtt SecretsKey.Arn
+  SecretsInstanceAccessPolicy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyName: !Sub key-${InfrastructureStorageStackName}-secrets-instance-access-policy
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - s3:GetObject
+              - s3:GetObjectVersion
+            Resource:
+              - !Sub arn:aws:s3:::${InfrastructureStorageStackName}-secrets/*
+  SecretsKey:
```

Comment:
I think (kind of guessing) if we ever needed to rotate this key, we would add another one to the template, and then change what the alias points at. So it might be good to give this a name like `SecretKeyMay17` or something? idk. maybe that's dumb